pkg/controller/batchrelease/batchrelease_plan_executor.go

```diff
@@ -197,7 +197,19 @@ func (r *Executor) progressBatches(workloadController workloads.WorkloadControll
 	klog.V(3).Infof("BatchRelease(%v) Canary Batch State Machine into '%s' state", r.releaseKey, status.CanaryStatus.CurrentBatchState)
 
 	switch status.CanaryStatus.CurrentBatchState {
-	case "", v1alpha1.UpgradingBatchState:
+
+	case "", v1alpha1.PreparingBatchState:
+		preparingDone, prepareErr := workloadController.PrepareOneBatch()
```

Comment:
can RolloutPhaseProgressing state transit back to RolloutPhaseProgressing, or to another state RolloutPhaseRolloutbackProgressing, PreparingRollback should be A batchrelease state, not A batch state